<launch>

    <!-- Arguments -->
    <arg name="url" default="ws://localhost:8080/ws"/>
    <arg name="realm" default="realm1"/>
    <arg name="robot_id" default="testbot"/>


    <node name="autobahn_wamp_client" pkg="ros_web_client" type="autobahn_wamp_client.py" output="screen"/>
 
    <rosparam param="pub2net_topics">['/clock', '/scan']</rosparam>
    <rosparam param="sub2net_topics">['/map', '/tf']</rosparam>

    <param name="webclient/robot_id" value="$(arg robot_id)" />
    <param name="webclient/url" value="$(arg url)" />
    <param name="webclient/realm" value="$(arg realm)" />
    <param name="webclient/data_domain" value="com.$(arg robot_id).data" />
    <param name="webclient/command_domain" value="com.$(arg robot_id).command" />


</launch>